site/en/docs/extensions/mv3/webusb/index.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: "layouts/doc-post.njk"
+title: "WebUSB in extensions"
+seoTitle: "Chrome Extensions: WebUSB"
+date: 2023-11-02
+description: The WebUSB API, which exposes non-standard Universal Serial Bus (USB) compatible devices to the web, is available in extensions.
+---
+
+The WebUSB API exposes non-standard Universal Serial Bus (USB) compatible devices to the web. This page describes aspects of the API that are particular to extensions. Refer to MDN for complete details of the [WebUSB API](https://developer.mozilla.org/docs/Web/API/WebUSB_API).
+
+## Availability in extensions
+
+Chrome 118 or later.
+
+## Permissions
+
+No manifest file permissions are required; however WebUSB triggers the browser's user permission flow.
+
+## Manifest
+
+No manifest keys are needed for this API.
+
+## Supporting contexts
+
+This API may be used in any context. The `WebUSB.requestDevice()` method cannot be used in extension service workers. See the next section for details.
```

Comment:
This could be more clear since later it says you can use it on the service worker via messaging. 
Could we rephrase it like it cannot be used directly in a sw, but there is a workaround in the next section?